core/client/components/character-name-text.js

```diff
@@ -1,34 +1,85 @@
 import Phaser from 'phaser';
 
-const style = {
+const nameStyle = {
   font: 'Verdana, "Times New Roman", Tahoma, serif',
   fontSize: 18,
   strokeColor: '#000',
   strokeSize: 3,
 };
 
+const baselineStyle = {
+  ...nameStyle,
+  fontSize: 12,
+};
+
 const defaultTint = 0xFFFFFF;
+const baselineOffset = 20;
 
-class CharacterNameText extends Phaser.GameObjects.Text {
-  constructor(scene, text, tintName) {
-    super(scene, 0, 0, text);
+class CharacterNameText {
```

Comment:
I wonder if we should not inherit from a `Phaser.GameObjects.Container` to have a "group" and have a relative transformation matrix.

Reply:
Thanks for the suggestion, you're right it is clearly better :)